base/applications/mspaint/registry.cpp

```diff
@@ -103,6 +103,13 @@ void RegistrySettings::Load()
         ReadFileHistory(files, _T("File3"), strFile3);
         ReadFileHistory(files, _T("File4"), strFile4);
     }
+
+    if (BMPWidth > 4096 || BMPHeight > 2304)
```

Comment:
These values look quite artificial. Why 2304 and not, say, 2305 or 6666 ?

Reply:
I think bigger than 4K resolution is too large.

https://en.wikipedia.org/wiki/4K_resolution

Reply:
How does Windows' paint behave, on machines that have a lot of RAM? (I'm asking because I would expect that this limitation is calculated at runtime, if there is even one...)

Reply:
I have no idea, but MS mspaint surely does fix the setting values if too large.

Reply:
![image](https://user-images.githubusercontent.com/2107452/147397463-69fd7b61-5fec-41db-9bc4-fefa104d0d5d.png)
I found the limitation is 5000x5000.

Reply:
Fixed.